js/api.js

```diff
@@ -0,0 +1,16 @@
+function fetchData(url) {
```

Comment:
This fetch function can be more dynamic by having two parameters so it can be used for other types of network requests. It was mentioned in our code review but the URL parameter can be dynamic as well